boards/pinetime/Makefile.include

```diff
@@ -0,0 +1,7 @@
+# for this board, we are using Segger's RTT as default terminal interface
+USEMODULE += stdio_rtt
+TERMPROG = $(RIOTTOOLS)/jlink/jlink.sh
+TERMFLAGS = term_rtt
+
+# use shared Makefile.include
+include $(RIOTBOARD)/common/nrf52xxxdk/Makefile.include
```

Comment:
Same as above, including `nrf52xxxdk`-files while this is not part of the DK development kits is a bit confusing.

Reply:
fixed